INCode/models.py

```diff
@@ -61,11 +61,9 @@ def load(self, file):
 
     def load_definition(self, declaration):
         translation_units = Index._gen_open(self.compilation_databases_.get_files())
-        candidates = Index._gen_search(declaration.get_name(), translation_units)
+        candidates = Index._gen_search(declaration.cursor_.spelling, translation_units)
```

Comment:
Please add tests:
- with identical function signatures spread over multiple lines in the H and CPP file
- with entire function signature on same line in H and function signature spread over multiple lines in CPP file

Reply:
done